[WIP] fix mistakes (#146)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-windows/overview.md

## What is AdGuard VPN for Windows?

> A VPN, acronym for "Virtual Private Network", is a service that makes your Internet connection safe and helps you stay anonymous online. How does it work? Every time you visit a website without using a VPN, your ISP sees it. The ISP knows who you are and what you're looking for, and it can collect and sell this data. In addition, the website you came to can also track your activity. When you enable a VPN application, it redirects your traffic through an encrypted tunnel to a remote VPN server, ensuring your privacy: the ISP doesn't know where to you sent a request, and the site doesn't know where you came from.

Reviewer: это кстати не ответ на вопрос What is AdGuard VPN for Windows

## What AdGuard VPN for Windows does

- Protects from network traffic interception (spoofing). AdGuard VPN creates an encrypted tunnel between your device and a remote server. All your Internet traffic passes through this tunnel, so your data is protected along the way. And thanks to [AdGuard's unique protocol](/general/adguard-vpn-protocol), you're guaranteed a fast and secure connection.

> - Masks your IP address. Your true IP address is the key to your personal data for cybercriminals. Your name, email address, phone number, credit card information can all fall into the hands of fraudsters if you don't hide your IP. With AdGuard VPN, as we said before, all your traffic goes through an encrypted tunnel and comes to the VPN server. The web server therefore registers the IP address of the endpoint of the tunnel, i.e. the VPN server, and thus not the device's real IP address.

Reviewer: не уверена что вся эта личная инфа связана с айпи

Reviewer: as we said before надо убрать

Reviewer: упростить последнее предложение, убрать therefore и thus

To export the list of exclusions from AdGuard VPN for Windows to your computer, click **Export exclusions**, select the folder where the list will be stored and click **Save**. An archive `exclusions.zip` with two `.txt` files will be downloaded, one for each of the lists — **General** and **Selective**. You can edit them by adding new exclusions or deleting old ones.

> To transfer the exclusion lists to another device, send the `.zip` file to that device. On the device where the exclusion lists are to be imported, open AdGuard VPN, click *Exclusions*, then *Import Exclusions* and select the received archive.

Reviewer: ```suggestion
On the destination device, open AdGuard VPN, click *Exclusions*, and select *Websites* or *Apps*. Click *Import exclusions* and select the received archive.
```

## Settings

![Settings](https://cdn.adguardvpn.com/content/release_notes/vpn/windows/v2.0/settings_en.png)

> The fourth tab contains sections that will help you customize the application. Let's look closer at two of them: **App settings** and **App exclusions**.

Reviewer: убрать App exclusions отсюда
